core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/RepairHistoryProviderImpl.java

```diff
@@ -89,16 +93,37 @@ public RepairHistoryProviderImpl(NodeResolver nodeResolver, CqlSession session,
         myClock = clock;
     }
 
+    /**
+     * Iterating over repair history in a table, based on time and filters.
```

Comment:
be more specific, "based on time" doesn't say anything. Maybe mention from-to and how these values are derived.